.github/workflows/go.yml

```diff
@@ -11,18 +11,17 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
         with:
           fetch-depth: 0
 
       - name: Set up Go
-        uses: actions/setup-go@v2
+        uses: actions/setup-go@v4
         with:
-          go-version: '1.20'
+          go-version: "1.21"
```

Comment:
Please include `check-latest: true` to ensure we are building with the latest vulnerability patches

Reply:
Thanks. Added this here and will make sure it also gets added to other repos with this change.